azure/v12.0.1/README.md

```diff
@@ -0,0 +1,15 @@
+## :zap:  Giant Swarm Release 12.0.1 for Azure :zap:
+
+**If you are upgrading from 12.0.0, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades external-dns app to v1.3.0.
+
+**Notes for future 12.x.x releases:**
+In order to proceed with upgrade, clusters must be using 11.4.0 or newer release version, otherwise the upgrade request will fail.
+Pay attention to the increased failure rate of requests during upgrade of the master node and include this information in release notes if the problem persists.
```

Comment:
```suggestion
Pay attention to the increased failure rate of requests during upgrade of the master node and include this information in release notes if the problem persists. For context, see [v12.0.0 release notes](https://github.com/giantswarm/releases/tree/master/azure/v12.0.0).
```

Reply:
@sslavic thank you! Makes sense. Just wanted to provide more context to the reader. Maybe linking to the v12.0.0 releases notes is enough?

Reply:
Will remove it since there's automation in place which will prevent the upgrade to skip multiple major or minor releases.